### ISSUE TITLE
Getting error: res.headers.raw is not a function

### ISSUE THREAD

**Issue by M-Nasab** [valeriangalliat/fetch-cookie#42]
I get this error when i use this library:

```
Uncaught (in promise) TypeError: res.headers.raw is not a function at fetchCookie (index.js?a60e:41)
```

**Comment by FabianTe** [comment]
This could happen because fetch-cookie assumes you use node-fetch v2 and may fail if you aren't.

What fetch implementation are you using in what environment?

**Comment by greggilbert** [comment]
FWIW I've got this happening right now, using `node-fetch` 2.6.1. Replicated in both Firefox and Safari.

**Comment by valeriangalliat** [comment]
Hey! I just published v2.0.0 with improved support for various `fetch` implementations!

`headers.raw` was pretty specific to node-fetch but now fetch-cookie also supports any `fetch` implementation compatible with the WHATWG Fetch standard API (as long as it exposes the `set-cookie` header)
